Src/AmrCore/AMReX_AmrMesh.cpp

```diff
@@ -361,6 +361,14 @@ AmrMesh::InitAmrMesh (int max_level_in, const Vector<int>& n_cell_in,
     {
         // chop up grids to have more grids than the number of procs
         pp.query("refine_grid_layout", refine_grid_layout);
+
+        // allow specification of dimensions that can be split
+        pp.queryarr("refine_grid_layout_dims", refine_grid_layout_dims);
```

Comment:
I think we should now make `refine_grid_layout` a local variable here not a member variable, and use its value to change the default value of `refine_grid_layout_dims`.  If both `refine_grid_layout` and `refine_grid_layout_dims` are in ParmParse, the latter will be used.  If only one is provided, that one will be used.

Reply:
I will work on fixing it this way. Thanks!

Reply:
Do we want to keep the `refine_grid_layout_x` yz names too?

Reply:
In the inputs.  But we don't need them to be variables.

Reply:
:+1: